lambdas/service/app.py

```diff
@@ -913,6 +963,14 @@ def repository_search(entity_type: str, item_id: str):
         logic. For example, `{"donorCount": {"within": [[1,5], [5,10]]}}`
         selects entities whose donor organism count falls within both
         ranges, i.e., is exactly 5.
+
+        The organismAge facet is special in that it contains two property keys
```

Comment:
Please create an issue about incorporating this special case into the schema so it can be enforced by generic validation. At the very least the `is` branch should allow unit-value dicts as facet values. Alternatively we could make this even more rigid by tying the value type to the facet name via field types. Every facet maps to a document path and every document path maps to a FieldType instance. You should quote me verbatim on the ticket.

Reply:
#2166 is the issue.